week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp

```diff
@@ -0,0 +1,103 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+
+struct Image {
+  double quality;
+  double freshness;
+  double rating;
+};
+
+struct Params {
+  double a;
+  double b;
+  double c;
+};
+
+
+class FunctionPart {
+ 
+  private:
```

Comment:
@Ypostolov Style problem? 
BTW why are you using private key here?

Reply:
Fixed.
Please check 14f1cb1cd14fd68b12659def58de3091dcaa7fe3